keras_cv/layers/preprocessing3d/group_points_by_bounding_boxes_test.py

```diff
@@ -26,6 +26,17 @@
 
 
 class GroupPointsByBoundingBoxesTest(tf.test.TestCase):
+    def test_get_params(self):
```

Comment:
one more thing -- can you update the training script to global_batch > 1, and verify it works?

Reply:
I will do this in next PR then